example/config/webpack/base.config.js

```diff
@@ -0,0 +1,259 @@
+const fs = require("fs-extra");
+const path = require("path");
+const webpack = require("webpack");
+const nodeExternals = require("webpack-node-externals");
+const HappyPack = require("happypack");
+const ProgressBarPlugin = require("progress-bar-webpack-plugin");
+
+const root = path.resolve(__dirname, "../..");
+
+const wildcatConfig = require("react-wildcat/src/utils/getWildcatConfig")(root);
+
+const {
+    generalSettings: {
+        env: {
+            __TEST__,
+            BABEL_ENV,
+            NODE_ENV = "development"
+        },
+        staticUrl
+    }
+} = wildcatConfig;
+
+const CACHE_DIR = ".cache";
+const CACHE_ENV_DIR = path.join(root, CACHE_DIR, NODE_ENV);
+fs.ensureDirSync(CACHE_ENV_DIR);
+
+const src = path.resolve(root, "src");
+
+const include = [
+    src // important for performance!
```

Comment:
Oh, this tells Babel to *only* compile files in the `src` directory. It makes sure you don't accidentally run `node_modules` through Babel.